app/javascript/application.js

```diff
@@ -1,18 +1,16 @@
 // Entry point for the build script in your package.json
 
-import 'jquery'
-import bootstrap from  "bootstrap/dist/js/bootstrap"
-global.bootstrap = bootstrap // Required for Blacklight 7 so it can manage the modals
+import "jquery"; // Blacklight 7 needs jQuery. Remove when we upgrade to Blacklight 8?
+import bootstrap from "bootstrap/dist/js/bootstrap";
+global.bootstrap = bootstrap; // Required for Blacklight 7 so it can manage the modals
 
-import 'blacklight-frontend/app/assets/javascripts/blacklight/blacklight'
-import "./controllers"
+import "blacklight-frontend/app/assets/javascripts/blacklight/blacklight";
+import "./controllers";
 
-import Argo from './argo'
+import Argo from "./argo";
 
 document.addEventListener("turbo:load", async () => {
-  await import('https://cdnjs.cloudflare.com/ajax/libs/free-jqgrid/4.15.5/jquery.jqgrid.src.js')
-
-  // Start argo after free-jqgrid has been loaded
-  new Argo().initialize()
-})
-import '@hotwired/turbo-rails'
+  // Start argo after Turbo has been loaded
+  new Argo().initialize();
+});
+import "@hotwired/turbo-rails";
```

Comment:
My editor did some normalization here. I intend to follow up later with more JS normalization and perhaps linting in Argo. I can roll these mechanical changes back if anyone cares.